test/quantization/fx/test_quantize_fx.py

```diff
@@ -447,6 +448,114 @@ def test_linear_tanh_not_fused_by_default(self):
             expected_node_list=expected_nodes,
             expected_node_occurrence=expected_occurrence)
 
+    def test_fuse_conv_bn_add_relu_onednn(self):
+        # conv - bn - add - relu is fused for onednn backend only
+        from torch.ao.quantization.backend_config import get_onednn_backend_config
+        expected_nodes = [
+            ns.call_module(nni.ConvAdd2d),
+        ]
+        expected_occurrence = {
+            ns.call_module(nni.ConvAdd2d): 1,
+            ns.call_module(nn.BatchNorm2d): 0,
+        }
+
+        options = itertools.product(
+            [True, False],  # with_bn
+            [False],  # with_relu
+            [True, False],  # conv in the left
+            [True, False],  # with_two_conv
+            [True, False],  # use_torch_add
+        )
+        for with_bn, with_relu, left_conv, two_conv, use_torch_add in options:
+            # test eval mode
+            m = ConvBnAddReluModel(
+                with_bn=with_bn,
+                with_relu=with_relu,
+                left_conv=left_conv,
+                two_conv=two_conv,
+                use_torch_add=use_torch_add).eval()
+
+            m = fuse_fx(m,
+                        backend_config=get_onednn_backend_config())
+            self.checkGraphModuleNodes(
+                m,
+                expected_node_list=expected_nodes,
+                expected_node_occurrence=expected_occurrence)
+
+    def test_fuse_conv_bn_add_relu_by_default(self):
+        options = itertools.product(
+            [True, False],  # with_bn
+            [False],  # with_relu
```

Comment:
same question for relu

Reply:
The support of relu has been added in the following up PRs.